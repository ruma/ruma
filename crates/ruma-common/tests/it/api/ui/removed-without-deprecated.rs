use ruma_common::{api::Metadata, metadata};

metadata! {
    method: GET,
    rate_limited: false,
    authentication: NoAuthentication,
    history: {
        unstable => "/a/path",
        1.1 => removed,
    }
}

pub struct Request;

fn main() {}
