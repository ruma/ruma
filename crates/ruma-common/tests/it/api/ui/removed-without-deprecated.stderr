error: no rules expected the token `removed`
 --> tests/it/api/ui/removed-without-deprecated.rs:9:16
  |
9 |         1.1 => removed,
  |                ^^^^^^^ no rules expected this token in macro call
  |
  = note: while trying to match sequence start
